.github/workflows/buid_and_release.yml

```diff
@@ -16,5 +16,20 @@ jobs:
     - uses: actions/checkout@v2
     - name: build
       run: make all
+    - name: Get current time
+      uses: josStorer/get-current-time@v2.0.2
+      id: current-time
+      with:
+        format: YYYY-MM-DD
     - name: Release
-      run: make deploy-release
```

Comment:
I left this command in the `makefile`. It's the way to make a release locally using the GitHub CLI.